### DISCUSSION TITLE
Welcome to xbase Discussions!

### DISCUSSION THREAD

**Discussion by kkharji** [kkharji/xbase#77] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by EastHell** [comment]
Hello, I'm a iOS developer, and want to try using `xbase` with [neovim](https://neovim.io/) and [SpaceVim](https://spacevim.org/) combination. But idk how to run `make install` and `lua require'xbase'.setup()` inside SpaceVim's `init.toml` file or `myspacevim.vim` fiile. Can you help me?

**Reply by kkharji** [reply]
Hey @EastHell, you can join xbase discord https://discord.gg/VHfG6Jdb and hopefully I will walk you through it

**Reply by anullog** [reply]
Hey @kkharji, could you please provide an updated invitation link since the one above has expired? Thanks!
